Applications/App/Resources/Settings/DefaultSettings.ini

```diff
@@ -4,6 +4,3 @@
 DontConfirmExit=1024
 DontConfirmRestart=1024
 DontShowDisclaimerMessage=1024
-
-[Modules]
-HomeModule=Home
```

Comment:
@jcfr why removing this? We want the custom `Home` module to be the homemodule

Reply:
Consider reading the commit message, this is explained there